Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -27,10 +27,8 @@ void CommonAsmRoutines::GenFifoWrite(int size)
   const void* start = GetCodePtr();
 
   // Assume value in RSCRATCH
```

Comment:
Oh dear. Why are we doing this here so complicated? Isn't it the same as:

ADD(32,M(&GPFifo::m_gatherPipeCount), Imm8(size >> 3));

And we CALL() this function instead of inlining ....